test/pylib/manager_client.py

```diff
@@ -83,6 +83,8 @@ async def _get(self, resource: str) -> aiohttp.ClientResponse:
 
     async def _get_text(self, resource: str) -> str:
         resp = await self._get(resource)
+        if resp.status != 200:
+            raise Exception(await resp.text())
```

Comment:
This commit causes tests to fail, you should move it after the commit which fixes the paths.

Reply:
Also, maybe we should do the check in the general `_get`

Reply:
Done in latest